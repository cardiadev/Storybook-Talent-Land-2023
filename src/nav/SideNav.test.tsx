// import * as React from "react";
// import { render, cleanup, RenderResult } from "@testing-library/react";
// import "@testing-library/jest-dom/extend-expect";
// import { ISideNavProps, SideNav } from "./SideNav";
// import { navItemsTop, navItemsBottom } from "../data/navItems";

// afterEach(cleanup);

// const setup = ({ open }: Partial<ISideNavProps>): ISetupSideNav => {
//   const utils = render(
//     <SideNav
//       className="relative z-50 h-screen -mt-20"
//       navItemsTop={navItemsTop}
//       navItemsBottom={navItemsBottom}
//       open={open ?? false}
//       setOpen={() => {}}
//       username="Veronica Woods"
//       email="veronica@example.com"
//     />,
//   );

//   const sideNav = utils.getByTestId("input") as HTMLElement;

//   return {
//     sideNav,
//     ...utils,
//   };
// };

// // TODO test filling in the searchString
// // TODO test opening/closing of SideNav
// describe("SideNav", () => {
//   it("Renders SideNav (open) correctly", () => {
//     const { asFragment } = setup({ open: true });

//     expect(asFragment()).toMatchSnapshot();
//   });

//   it("Renders SideNav (closed) correctly", () => {
//     const { asFragment } = setup({ open: false });

//     expect(asFragment()).toMatchSnapshot();
//   });

//   // it("Filling in SearchInput works", () => {
//   //   const handleChange = jest.fn();

//   //   const { asFragment } = setup({open: true});
//   //   const value = "veronica@example.com";

//   //   fireEvent.change(input, { target: { value } });

//   //   expect(handleChange).toHaveBeenCalledWith(value);
//   // });
// });
