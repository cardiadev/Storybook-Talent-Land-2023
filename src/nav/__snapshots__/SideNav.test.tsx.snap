// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideNav Renders SideNav (closed) correctly 1`] = `
<DocumentFragment>
  <div
    class="py-6 flex flex-col flex-grow bg-white dark:bg-gray-900 transform ease-out duration-100 overflow-y-auto xs:overflow-y-visible w-24 border-r border-gray-300 dark:border-opacity-20 relative z-50 h-screen -mt-20"
  >
    <div
      class="flex items-center w-full mb-8 ml-9"
    >
      <img
        alt="logo"
        class="w-8 h-8 select-none"
        src="https://res.cloudinary.com/tailwindcss/image/upload/v1634916081/Logo_Icon_dq276z.png"
      />
      <p
        class="text-xl font-medium ml-2.5 whitespace-nowrap select-none opacity-0 hidden text-black dark:text-white"
      >
        My Travel App
      </p>
    </div>
    <div
      class="mb-8 mx-9"
    >
      <div
        class="h-11"
      />
    </div>
    <div
      class="flex flex-col justify-between h-full ml-5 w-70 w-14"
    >
      <ul
        class="flex flex-col space-y-1"
      >
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group ml-2 px-2 w-10"
            id="headlessui-popover-button-17"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M3 9l9-7 9 7v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2z"
              />
              <polyline
                points="9 22 9 12 15 12 15 22"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 opacity-0 hidden group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Home
            </p>
          </li>
        </div>
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group ml-2 px-2 w-10"
            id="headlessui-popover-button-19"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <line
                x1="18"
                x2="18"
                y1="20"
                y2="10"
              />
              <line
                x1="12"
                x2="12"
                y1="20"
                y2="4"
              />
              <line
                x1="6"
                x2="6"
                y1="20"
                y2="14"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 opacity-0 hidden group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Dashboard
            </p>
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white transform duration-100 ease-out opacity-0 hidden text-gray-400"
              fill="none"
              height="20"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polyline
                points="6 9 12 15 18 9"
              />
            </svg>
          </li>
        </div>
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group ml-2 px-2 w-10"
            id="headlessui-popover-button-21"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polygon
                points="12 2 2 7 12 12 22 7 12 2"
              />
              <polyline
                points="2 17 12 22 22 17"
              />
              <polyline
                points="2 12 12 17 22 12"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 opacity-0 hidden group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Projects
            </p>
          </li>
        </div>
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group ml-2 px-2 w-10"
            id="headlessui-popover-button-23"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polyline
                points="9 11 12 14 22 4"
              />
              <path
                d="M21 12v7a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h11"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 opacity-0 hidden group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Tasks
            </p>
          </li>
        </div>
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group ml-2 px-2 w-10"
            id="headlessui-popover-button-25"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M4 15s1-1 4-1 5 2 8 2 4-1 4-1V3s-1 1-4 1-5-2-8-2-4 1-4 1z"
              />
              <line
                x1="4"
                x2="4"
                y1="22"
                y2="15"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 opacity-0 hidden group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Reporting
            </p>
          </li>
        </div>
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group ml-2 px-2 w-10"
            id="headlessui-popover-button-27"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M21 10c0 7-9 13-9 13s-9-6-9-13a9 9 0 0 1 18 0z"
              />
              <circle
                cx="12"
                cy="10"
                r="3"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 opacity-0 hidden group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Rentals
            </p>
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white transform duration-100 ease-out opacity-0 hidden text-gray-400"
              fill="none"
              height="20"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polyline
                points="6 9 12 15 18 9"
              />
            </svg>
          </li>
        </div>
      </ul>
      <ul
        class="flex flex-col mb-6 space-y-1"
      >
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group ml-2 px-2 w-10"
            id="headlessui-popover-button-29"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                cx="12"
                cy="12"
                r="3"
              />
              <path
                d="M19.4 15a1.65 1.65 0 0 0 .33 1.82l.06.06a2 2 0 0 1 0 2.83 2 2 0 0 1-2.83 0l-.06-.06a1.65 1.65 0 0 0-1.82-.33 1.65 1.65 0 0 0-1 1.51V21a2 2 0 0 1-2 2 2 2 0 0 1-2-2v-.09A1.65 1.65 0 0 0 9 19.4a1.65 1.65 0 0 0-1.82.33l-.06.06a2 2 0 0 1-2.83 0 2 2 0 0 1 0-2.83l.06-.06a1.65 1.65 0 0 0 .33-1.82 1.65 1.65 0 0 0-1.51-1H3a2 2 0 0 1-2-2 2 2 0 0 1 2-2h.09A1.65 1.65 0 0 0 4.6 9a1.65 1.65 0 0 0-.33-1.82l-.06-.06a2 2 0 0 1 0-2.83 2 2 0 0 1 2.83 0l.06.06a1.65 1.65 0 0 0 1.82.33H9a1.65 1.65 0 0 0 1-1.51V3a2 2 0 0 1 2-2 2 2 0 0 1 2 2v.09a1.65 1.65 0 0 0 1 1.51 1.65 1.65 0 0 0 1.82-.33l.06-.06a2 2 0 0 1 2.83 0 2 2 0 0 1 0 2.83l-.06.06a1.65 1.65 0 0 0-.33 1.82V9a1.65 1.65 0 0 0 1.51 1H21a2 2 0 0 1 2 2 2 2 0 0 1-2 2h-.09a1.65 1.65 0 0 0-1.51 1z"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 opacity-0 hidden group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Settings
            </p>
          </li>
        </div>
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group ml-2 px-2 w-10"
            id="headlessui-popover-button-31"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out -rotate-180 text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <line
                x1="19"
                x2="5"
                y1="12"
                y2="12"
              />
              <polyline
                points="12 19 5 12 12 5"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 opacity-0 hidden group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Close Navbar
            </p>
          </li>
        </div>
      </ul>
    </div>
    <hr
      class="ml-5 border-t border-gray-300 dark:border-opacity-20 w-14"
    />
    <div
      class="flex mt-6 w-70 ml-7"
    >
      <img
        alt="avatar"
        class="w-10 h-10 rounded-full cursor-pointer select-none"
        src="https://res.cloudinary.com/tailwindcss/image/upload/v1634915122/demoAvatar_jooj6y.png"
      />
    </div>
  </div>
</DocumentFragment>
`;

exports[`SideNav Renders SideNav (open) correctly 1`] = `
<DocumentFragment>
  <div
    class="py-6 flex flex-col flex-grow bg-white dark:bg-gray-900 transform ease-out duration-100 overflow-y-auto xs:overflow-y-visible w-80 border-r border-gray-300 dark:border-opacity-20 relative z-50 h-screen -mt-20"
  >
    <div
      class="flex items-center w-full mb-8 ml-9"
    >
      <img
        alt="logo"
        class="w-8 h-8 select-none"
        src="https://res.cloudinary.com/tailwindcss/image/upload/v1634916081/Logo_Icon_dq276z.png"
      />
      <p
        class="text-xl font-medium ml-2.5 whitespace-nowrap select-none text-black dark:text-white"
      >
        My Travel App
      </p>
    </div>
    <div
      class="mb-8 mx-9"
    >
      <div
        class="relative"
      >
        <div
          class="h-11 shadow-sm rounded-lg w-full absolute flex items-center justify-between px-3.5 pointer-events-none"
        >
          <svg
            class="text-gray-500"
            fill="none"
            height="1em"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              cx="11"
              cy="11"
              r="8"
            />
            <line
              x1="21"
              x2="16.65"
              y1="21"
              y2="16.65"
            />
          </svg>
        </div>
        <input
          aria-label="input"
          class="w-full select-none text-gray-900 dark:text-white text-md border h-11 pl-9 rounded-lg border-gray-300 dark:border-gray-500 focus:ring-4 focus:border-primary-300 dark:focus:border-gray-100 focus:ring-primary-100 dark:focus:ring-gray-100 dark:focus:ring-opacity-20 bg-white dark:bg-gray-800"
          placeholder="Search"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      class="flex flex-col justify-between h-full ml-5 w-70"
    >
      <ul
        class="flex flex-col space-y-1"
      >
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group px-4"
            id="headlessui-popover-button-1"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M3 9l9-7 9 7v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2z"
              />
              <polyline
                points="9 22 9 12 15 12 15 22"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Home
            </p>
          </li>
        </div>
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group px-4"
            id="headlessui-popover-button-3"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <line
                x1="18"
                x2="18"
                y1="20"
                y2="10"
              />
              <line
                x1="12"
                x2="12"
                y1="20"
                y2="4"
              />
              <line
                x1="6"
                x2="6"
                y1="20"
                y2="14"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Dashboard
            </p>
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white transform duration-100 ease-out text-gray-400"
              fill="none"
              height="20"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polyline
                points="6 9 12 15 18 9"
              />
            </svg>
          </li>
        </div>
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group px-4"
            id="headlessui-popover-button-5"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polygon
                points="12 2 2 7 12 12 22 7 12 2"
              />
              <polyline
                points="2 17 12 22 22 17"
              />
              <polyline
                points="2 12 12 17 22 12"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Projects
            </p>
          </li>
        </div>
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group px-4"
            id="headlessui-popover-button-7"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polyline
                points="9 11 12 14 22 4"
              />
              <path
                d="M21 12v7a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h11"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Tasks
            </p>
          </li>
        </div>
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group px-4"
            id="headlessui-popover-button-9"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M4 15s1-1 4-1 5 2 8 2 4-1 4-1V3s-1 1-4 1-5-2-8-2-4 1-4 1z"
              />
              <line
                x1="4"
                x2="4"
                y1="22"
                y2="15"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Reporting
            </p>
          </li>
        </div>
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group px-4"
            id="headlessui-popover-button-11"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M21 10c0 7-9 13-9 13s-9-6-9-13a9 9 0 0 1 18 0z"
              />
              <circle
                cx="12"
                cy="10"
                r="3"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Rentals
            </p>
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white transform duration-100 ease-out text-gray-400"
              fill="none"
              height="20"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polyline
                points="6 9 12 15 18 9"
              />
            </svg>
          </li>
        </div>
      </ul>
      <ul
        class="flex flex-col mb-6 space-y-1"
      >
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group px-4"
            id="headlessui-popover-button-13"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                cx="12"
                cy="12"
                r="3"
              />
              <path
                d="M19.4 15a1.65 1.65 0 0 0 .33 1.82l.06.06a2 2 0 0 1 0 2.83 2 2 0 0 1-2.83 0l-.06-.06a1.65 1.65 0 0 0-1.82-.33 1.65 1.65 0 0 0-1 1.51V21a2 2 0 0 1-2 2 2 2 0 0 1-2-2v-.09A1.65 1.65 0 0 0 9 19.4a1.65 1.65 0 0 0-1.82.33l-.06.06a2 2 0 0 1-2.83 0 2 2 0 0 1 0-2.83l.06-.06a1.65 1.65 0 0 0 .33-1.82 1.65 1.65 0 0 0-1.51-1H3a2 2 0 0 1-2-2 2 2 0 0 1 2-2h.09A1.65 1.65 0 0 0 4.6 9a1.65 1.65 0 0 0-.33-1.82l-.06-.06a2 2 0 0 1 0-2.83 2 2 0 0 1 2.83 0l.06.06a1.65 1.65 0 0 0 1.82.33H9a1.65 1.65 0 0 0 1-1.51V3a2 2 0 0 1 2-2 2 2 0 0 1 2 2v.09a1.65 1.65 0 0 0 1 1.51 1.65 1.65 0 0 0 1.82-.33l.06-.06a2 2 0 0 1 2.83 0 2 2 0 0 1 0 2.83l-.06.06a1.65 1.65 0 0 0-.33 1.82V9a1.65 1.65 0 0 0 1.51 1H21a2 2 0 0 1 2 2 2 2 0 0 1-2 2h-.09a1.65 1.65 0 0 0-1.51 1z"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Settings
            </p>
          </li>
        </div>
        <div
          class="relative"
        >
          <li
            aria-expanded="false"
            class="navItem group px-4"
            id="headlessui-popover-button-15"
          >
            <svg
              class="group-hover:text-primary-600 dark:group-hover:text-white dark:text-white transform duration-100 ease-out text-gray-900"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <line
                x1="19"
                x2="5"
                y1="12"
                y2="12"
              />
              <polyline
                points="12 19 5 12 12 5"
              />
            </svg>
            <p
              class="text-md font-medium flex-grow ml-3 group-hover:text-primary-600 dark:group-hover:text-white text-gray-800 dark:text-white"
            >
              Close Navbar
            </p>
          </li>
        </div>
      </ul>
    </div>
    <hr
      class="ml-5 border-t border-gray-300 dark:border-opacity-20 w-70"
    />
    <div
      class="flex mt-6 w-70 ml-7"
    >
      <img
        alt="avatar"
        class="w-10 h-10 rounded-full cursor-pointer select-none"
        src="https://res.cloudinary.com/tailwindcss/image/upload/v1634915122/demoAvatar_jooj6y.png"
      />
      <div
        class="flex justify-between ml-3"
      >
        <div
          class="select-none"
        >
          <p
            class="text-sm font-medium text-gray-700 dark:text-white"
          >
            Veronica Woods
          </p>
          <p
            class="text-sm font-normal text-gray-500"
          >
            veronica@example.com
          </p>
        </div>
        <svg
          class="ml-6 text-gray-400 cursor-pointer"
          fill="none"
          height="24"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9 21H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h4"
          />
          <polyline
            points="16 17 21 12 16 7"
          />
          <line
            x1="21"
            x2="9"
            y1="12"
            y2="12"
          />
        </svg>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
